<!doctype html><html lang=en><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta name=robots content="noodp"><title class=pjax-title>Felmlee 2013 : Mechanism-based pharmacodynamic modeling - Reading List</title><meta name=Description content="Reading Note of Wen-Wei"><meta property="og:title" content="Felmlee 2013 : Mechanism-based pharmacodynamic modeling"><meta property="og:description" content="Sciwheel1, PMC3684160"><meta property="og:type" content="article"><meta property="og:url" content="https://sosiristseng.github.io/readlist/2020/10/felmlee-2013/"><meta property="og:image" content="https://sosiristseng.github.io/readlist/"><meta property="article:section" content="posts"><meta property="article:published_time" content="2020-10-22T23:52:58+08:00"><meta property="article:modified_time" content="2022-07-07T21:00:38+08:00"><meta property="og:site_name" content="Reading List"><meta property="og:see_also" content="https://sosiristseng.github.io/readlist/2022/07/tran-2015/"><meta property="og:see_also" content="https://sosiristseng.github.io/readlist/2022/07/takeuchi-2020/"><meta property="og:see_also" content="https://sosiristseng.github.io/readlist/2022/07/tewari-2012/"><meta property="og:see_also" content="https://sosiristseng.github.io/readlist/2021/01/gregg-2016/"><meta property="og:see_also" content="https://sosiristseng.github.io/readlist/2020/10/zorov-2014/"><meta property="og:see_also" content="https://sosiristseng.github.io/readlist/2020/10/yang-2018/"><meta name=twitter:card content="summary_large_image"><meta name=twitter:image content="https://sosiristseng.github.io/readlist/"><meta name=twitter:title content="Felmlee 2013 : Mechanism-based pharmacodynamic modeling"><meta name=twitter:description content="Sciwheel1, PMC3684160"><meta name=application-name content="Reading Notes"><meta name=apple-mobile-web-app-title content="Reading Notes"><meta name=theme-color content="#f8f8f8"><meta name=msapplication-TileColor content="#da532c"><link rel="shortcut icon" type=image/x-icon href=/favicon.ico><link rel=icon type=image/png sizes=32x32 href=/favicon-32x32.png><link rel=icon type=image/png sizes=16x16 href=/favicon-16x16.png><link rel=apple-touch-icon sizes=180x180 href=/apple-touch-icon.png><link rel=mask-icon href=/safari-pinned-tab.svg color=#5bbad5><link rel=manifest href=/site.webmanifest><link rel=canonical href=https://sosiristseng.github.io/readlist/2020/10/felmlee-2013/><link rel=prev href=https://sosiristseng.github.io/readlist/2020/10/ellinwood-2017/><link rel=next href=https://sosiristseng.github.io/readlist/2020/10/fernandezchas-2018/><link rel=stylesheet href=/readlist/lib/normalize/normalize.min.css><link rel=stylesheet href=/readlist/css/style.min.css><link rel=preload as=style onload='this.onload=null,this.rel="stylesheet"' href=/readlist/lib/fontawesome-free/all.min.css><noscript><link rel=stylesheet href=/readlist/lib/fontawesome-free/all.min.css></noscript><link rel=preload as=style onload='this.onload=null,this.rel="stylesheet"' href=/readlist/lib/animate/animate.min.css><noscript><link rel=stylesheet href=/readlist/lib/animate/animate.min.css></noscript><script type=application/ld+json>{"@context":"http://schema.org","@type":"BlogPosting","headline":"Felmlee 2013 : Mechanism-based pharmacodynamic modeling","inLanguage":"en","mainEntityOfPage":{"@type":"WebPage","@id":"https:\/\/sosiristseng.github.io\/readlist\/2020\/10\/felmlee-2013\/"},"genre":"posts","keywords":"quantitative systems pharmacology, enzyme kinetics","wordcount":590,"url":"https:\/\/sosiristseng.github.io\/readlist\/2020\/10\/felmlee-2013\/","datePublished":"2020-10-22T23:52:58+08:00","dateModified":"2022-07-07T21:00:38+08:00","license":"This work is licensed under a Creative Commons Attribution-NonCommercial 4.0 International License.","publisher":{"@type":"Organization","name":"xxxx"},"author":{"@type":"Person","name":"Felmlee and others"},"description":""}</script></head><body header-desktop=fixed header-mobile=auto><script type=text/javascript>function setTheme(e){document.body.setAttribute("theme",e),document.documentElement.style.setProperty("color-scheme",e==="light"?"light":"dark")}function saveTheme(e){window.localStorage&&localStorage.setItem("theme",e)}function getMeta(e){const t=document.getElementsByTagName("meta");for(let n=0;n<t.length;n++)if(t[n].getAttribute("name")===e)return t[n];return""}if(window.localStorage&&localStorage.getItem("theme")){let e=localStorage.getItem("theme");e==="light"||e==="dark"||e==="black"?setTheme(e):setTheme(window.matchMedia&&window.matchMedia("(prefers-color-scheme: dark)").matches?"dark":"light")}else"auto"==="light"||"auto"==="dark"||"auto"==="black"?(setTheme("auto"),saveTheme("auto")):(saveTheme("auto"),setTheme(window.matchMedia&&window.matchMedia("(prefers-color-scheme: dark)").matches?"dark":"light"));let metaColors={light:"#f8f8f8",dark:"#252627",black:"#000000"};getMeta("theme-color").content=metaColors[document.body.getAttribute("theme")]</script><div id=back-to-top></div><div id=mask></div><div class=wrapper><header class=desktop id=header-desktop><div class=header-wrapper><div class=header-title><a href=/readlist/ title="Reading List"><span class=header-title-pre><i class='fas fa-edit fa-fw'></i></span>Reading List</a></div><div class=menu><div class=menu-inner><a class=menu-item href=/readlist/posts/>All Reading Notes </a><a class=menu-item href=/readlist/tags/>Tags </a><a class=menu-item href=/readlist/categories/>Categories </a><a class=menu-item href=/readlist/series/>Series </a><a class=menu-item href=https://github.com/sosiristseng/readlist title=GitHub rel="noopener noreffer" target=_blank><i class='fab fa-github fa-fw'></i> </a><span class="menu-item delimiter"></span><span class="menu-item search" id=search-desktop>
<input type=text placeholder=Search id=search-input-desktop>
<a href=# onclick=return!1 class="search-button search-toggle" id=search-toggle-desktop title=Search><i class="fas fa-search fa-fw"></i></a>
<a href=# onclick=return!1 class="search-button search-clear" id=search-clear-desktop title=Clear><i class="fas fa-times-circle fa-fw"></i></a>
<span class="search-button search-loading" id=search-loading-desktop><i class="fas fa-spinner fa-fw fa-spin"></i></span>
</span><a href=# onclick=return!1 class="menu-item theme-switch" title="Switch Theme"><i class="fas fa-adjust fa-fw"></i></a></div></div></div></header><header class=mobile id=header-mobile><div class=header-container><div class=header-wrapper><div class=header-title><a href=/readlist/ title="Reading List"><span class=header-title-pre><i class='fas fa-edit fa-fw'></i></span>Reading List</a></div><div class=menu-toggle id=menu-toggle-mobile><span></span><span></span><span></span></div></div><div class=menu id=menu-mobile><div class=search-wrapper><div class="search mobile" id=search-mobile><input type=text placeholder=Search id=search-input-mobile>
<a href=# onclick=return!1 class="search-button search-toggle" id=search-toggle-mobile title=Search><i class="fas fa-search fa-fw"></i></a>
<a href=# onclick=return!1 class="search-button search-clear" id=search-clear-mobile title=Clear><i class="fas fa-times-circle fa-fw"></i></a>
<span class="search-button search-loading" id=search-loading-mobile><i class="fas fa-spinner fa-fw fa-spin"></i></span></div><a href=# onclick=return!1 class=search-cancel id=search-cancel-mobile>Cancel</a></div><a class=menu-item href=/readlist/posts/ title>All Reading Notes</a><a class=menu-item href=/readlist/tags/ title>Tags</a><a class=menu-item href=/readlist/categories/ title>Categories</a><a class=menu-item href=/readlist/series/ title>Series</a><a class=menu-item href=https://github.com/sosiristseng/readlist title=GitHub rel="noopener noreffer" target=_blank><i class='fab fa-github fa-fw'></i></a><a href=# onclick=return!1 class="menu-item theme-switch" title="Switch Theme">
<i class="fas fa-adjust fa-fw"></i></a></div></div></header><div class="search-dropdown desktop"><div id=search-dropdown-desktop></div></div><div class="search-dropdown mobile"><div id=search-dropdown-mobile></div></div><main class=main><div class=container><div class=toc id=toc-auto><h2 class=toc-title>Contents</h2><div class=toc-content id=toc-content-auto><nav id=TableOfContents><ul><li><a href=#introduction>Introduction</a></li><li><a href=#modeling-requirements>Modeling Requirements</a><ul><li><a href=#simple-direct-effect-models>Simple Direct Effect Models</a></li><li><a href=#biophase-distribution>Biophase Distribution</a></li><li><a href=#indirect-response-models>Indirect Response Models</a><ul><li><a href=#model-1>Model 1</a></li><li><a href=#model-2>Model 2</a></li><li><a href=#model-3>Model 3</a></li><li><a href=#model-4>Model 4</a></li></ul></li><li><a href=#signal-transduction-models>Signal Transduction Models</a></li><li><a href=#irreversible-effect-models>Irreversible Effect Models</a></li><li><a href=#more-complex-models>More Complex Models</a><ul><li><a href=#the-time-course-of-paraoxon-inactivation-of-in-vitro-whole-blood-cholinesterase-wbche>The time-course of paraoxon inactivation of in vitro whole blood cholinesterase (WBChE)</a></li></ul></li></ul></li><li><a href=#prospects>Prospects</a></li></ul></nav></div></div><script>document.getElementsByTagName("main")[0].setAttribute("pageStyle","normal")</script><script>document.getElementsByTagName("main")[0].setAttribute("autoTOC","true")</script><article class="page single"><h1 class="single-title animate__animated animate__flipInX">Felmlee 2013 : Mechanism-based pharmacodynamic modeling</h1><div class=post-meta><div class=post-meta-line><span class=post-author><i class="author fas fa-user-circle fa-fw"></i><a href=https://github.com/sosiristseng title=Author target=_blank rel="noopener noreffer author" class=author>Felmlee and others</a>
</span>&nbsp;<span class=post-category>included in </span>&nbsp;<span class=post-category>category <a href=/readlist/categories/review/><i class="far fa-folder fa-fw"></i>Review</a></span>&nbsp;<span class=post-category>and</span>&nbsp;<span class=post-series>series <a href=/readlist/series/heart-modeling/><i class="far fa-list-alt fa-fw"></i>Heart modeling</a></span></div><div class=post-meta-line><i class="far fa-calendar-alt fa-fw"></i>&nbsp;<time datetime=2020-10-22>2020-10-22</time>&nbsp;<i class="far fa-edit fa-fw"></i>&nbsp;<time datetime=2022-07-07>2022-07-07</time>&nbsp;<i class="fas fa-pencil-alt fa-fw"></i>&nbsp;590 words&nbsp;
<i class="far fa-clock fa-fw"></i>&nbsp;3 minutes&nbsp;</div></div><div class="details series-nav open"><div class="details-summary series-title"><span>Series - Heart modeling</span>
<span><i class="details-icon fas fa-angle-right"></i></span></div><div class="details-content series-content"><nav><ul><li><a href=/readlist/2022/07/tran-2015/>Tran 2015: Regulation of cardiac cellular bioenergetics: mechanisms and consequences</a></li><li><a href=/readlist/2022/07/takeuchi-2020/>Takeuchi 2020: Integration of mitochondrial energetics in heart with mathematical modelling</a></li><li><a href=/readlist/2022/07/tewari-2012/>Tewari 2012: A biophysical model of the mitochondrial ATP-Mg/P(i) carrier</a></li><li><a href=/readlist/2021/01/gregg-2016/>Gregg 2016 : Pancreatic β-Cells From Mice Offset Age-Associated Mitochondrial Deficiency With Reduced KATP Channel Activity</a></li><li><a href=/readlist/2020/10/zorov-2014/>Zorov 2014 : Mitochondrial Reactive Oxygen Species (ROS) and ROS-Induced ROS Release</a></li><li><a href=/readlist/2020/10/yang-2018/>Yang 2018 : Mitochondrial‐Mediated Oxidative Ca2+/Calmodulin‐Dependent Kinase II Activation Induces Early Afterdepolarizations in Guinea Pig Cardiomyocytes: An In Silico Study</a></li><li><a href=/readlist/2020/10/zhu-2016/>Zhu 2016 : Doxorubicin redox biology: redox cycling, topoisomerase inhibition, and oxidative stress</a></li><li><a href=/readlist/2020/10/zhou-2014/>Zhou 2014 : Effects of Regional Mitochondrial Depolarization on Electrical Propagation: Implications for Arrhythmogenesis</a></li><li><a href=/readlist/2020/10/zhou-2012/>Zhou 2012 : Cardiac mitochondrial network excitability: insights from computational analysis</a></li><li><a href=/readlist/2020/10/zhou-2009/>Zhou 2009 : Modeling Cardiac Action Potential Shortening Driven by Oxidative Stress-Induced Mitochondrial Oscillations in Guinea Pig Cardiomyocytes</a></li><li><a href=/readlist/2020/10/zhang-2016/>Zhang 2016 : Multi-scale Modeling of the Cardiovascular System: Disease Development, Progression, and Clinical Intervention</a></li><li><a href=/readlist/2020/10/yang-2016/>Yang 2016 : In silico prediction of drug therapy in catecholaminergic polymorphic ventricular tachycardia</a></li><li><a href=/readlist/2020/10/yang-2014/>Yang 2014 : Mitochondria and arrhythmias</a></li><li><a href=/readlist/2020/10/wilson-2014/>Wilson 2014 : Mitochondrial cytochrome c oxidase: mechanism of action and role in regulating oxidative phosphorylation</a></li><li><a href=/readlist/2020/10/wei-2012/>Wei 2012 : Dynamics of matrix-free Ca2+ in cardiac mitochondria: two components of Ca2+ uptake and role of phosphate buffering</a></li><li><a href=/readlist/2020/10/wei-2011/>Wei 2011 : Mitochondrial energetics, pH regulation, and ion dynamics: a computational-experimental approach</a></li><li><a href=/readlist/2020/10/wang-2014/>Wang 2014 : Fast acceleration of 2D wave propagation simulations using modern computational accelerators</a></li><li><a href=/readlist/2020/10/vinogradov-2016/>Vinogradov 2016 : Oxidation of NADH and ROS production by respiratory complex I</a></li><li><a href=/readlist/2020/10/trayanova-2012/>Trayanova 2012 : Computational cardiology: how computer simulations could be used to develop new therapies and advance existing ones</a></li><li><a href=/readlist/2020/10/tran-2010/>Tran 2010 : A metabolite-sensitive, thermodynamically constrained model of cardiac cross-bridge cycling: implications for force development during ischemia</a></li><li><a href=/readlist/2020/10/tran-2009/>Tran 2009 : A thermodynamic model of the cardiac sarcoplasmic/endoplasmic Ca(2+) (SERCA) pump</a></li><li><a href=/readlist/2020/10/tomek-2018/>Tomek 2018 : Modulation of Cardiac Alternans by Altered Sarcoplasmic Reticulum Calcium Release: A Simulation Study</a></li><li><a href=/readlist/2020/10/smith-2004/>Smith 2004 : Development of models of active ion transport for whole-cell modelling: cardiac sodium-potassium pump as a case study</a></li><li><a href=/readlist/2020/10/shim-2017/>Shim 2017 : Mechanistic systems modeling to improve understanding and prediction of cardiotoxicity caused by targeted cancer therapeutics</a></li><li><a href=/readlist/2020/10/sazanov-1994/>Sazanov 1994 : Proton-translocating transhydrogenase and NAD- and NADP-linked isocitrate dehydrogenases operate in a substrate cycle which contributes to fine regulation of the tricarboxylic acid cycle activity in mitochondria</a></li><li><a href=/readlist/2020/10/sarkar-2010/>Sarkar 2010 : Regression analysis for constraining free parameters in electrophysiological models of cardiac cells</a></li><li><a href=/readlist/2020/10/sachetto-2018/>Sachetto 2018 : Performance evaluation of GPU parallelization, space-time adaptive algorithms, and their combination for simulating cardiac electrophysiology</a></li><li><a href=/readlist/2020/10/rocha-2011/>Rocha 2011 : Accelerating cardiac excitation spread simulations using graphics processing units</a></li><li><a href=/readlist/2020/10/roberts-2012/>Roberts 2012 : Computational approaches to understand cardiac electrophysiology and arrhythmias</a></li><li><a href=/readlist/2020/10/robb-2018/>Robb 2018 : Control of mitochondrial superoxide production by reverse electron transport at complex I</a></li><li><a href=/readlist/2020/10/qu-2014/>Qu 2014 : Nonlinear and stochastic dynamics in the heart</a></li><li><a href=/readlist/2020/10/plank-2008/>Plank 2008 : From mitochondrial ion channels to arrhythmias in the heart: computational techniques to bridge the spatio-temporal scales</a></li><li><a href=/readlist/2020/10/pereira-2016/>Pereira 2016 : Computational Models of Reactive Oxygen Species as Metabolic Byproducts and Signal-Transduction Modulators</a></li><li><a href=/readlist/2020/10/pathmanathan-2018/>Pathmanathan 2018 : Validation and Trustworthiness of Multiscale Models of Cardiac Electrophysiology</a></li><li><a href=/readlist/2020/10/onal-2014/>Onal 2014 : Modeling CaMKII in cardiac physiology: from molecule to tissue</a></li><li><a href=/readlist/2020/10/ohara-2012/>Ohara 2012 : Quantitative comparison of cardiac ventricular myocyte electrophysiology and response to drugs in human and nonhuman species</a></li><li><a href=/readlist/2020/10/octavia-2012/>Octavia 2012 : Doxorubicin-induced cardiomyopathy: From molecular mechanisms to therapeutic strategies</a></li><li><a href=/readlist/2020/10/nobile-2016/>Nobile 2016 : Graphics processing units in bioinformatics, computational biology and systems biology</a></li><li><a href=/readlist/2020/10/nivala-2010/>Nivala 2010 : Linking flickering to waves and whole-cell oscillations in a mitochondrial network model</a></li><li><a href=/readlist/2020/10/mustroph-2014/>Mustroph 2014 : CaMKII regulation of cardiac K channels</a></li><li><a href=/readlist/2020/10/mullins-2013/>Mullins 2013 : A Mathematical Model of the Mouse Ventricular Myocyte Contraction</a></li><li><a href=/readlist/2020/10/mirams-2018/>Mirams 2018 : Uncertainty and variability in computational and mathematical models of cardiac physiology</a></li><li><a href=/readlist/2020/10/michailova-2005/>Michailova 2005 : Modeling regulation of cardiac KATP and L-type Ca2+ currents by ATP, ADP, and Mg</a></li><li><a href=/readlist/2020/10/metelkin-2009/>Metelkin 2009 : Modeling of ATP–ADP steady‐state exchange rate mediated by the adenine nucleotide translocase in isolated mitochondria</a></li><li><a href=/readlist/2020/10/maillet-2016/>Maillet 2016 : Modeling Doxorubicin-Induced Cardiotoxicity in Human Pluripotent Stem Cell Derived-Cardiomyocytes</a></li><li><a href=/readlist/2020/10/lopezperez-2015/>Lopezperez 2015 : Three-dimensional cardiac computational modelling: methods, features and applications</a></li><li><a href=/readlist/2020/10/li-2015/>Li 2015 : Mitochondria-derived ROS bursts disturb Ca cycling and induce abnormal automaticity in guinea pig cardiomyocytes: a theoretical study</a></li><li><a href=/readlist/2020/10/li-2014/>Li 2014 : Inhibiting Na+/K+ ATPase can impair mitochondrial energetics and induce abnormal Ca2+ cycling and automaticity in guinea pig cardiomyocytes</a></li><li><a href=/readlist/2020/10/li-2012/>Li 2012 : Mechanisms by which Cytoplasmic Calcium Wave Propagation and Alternans Are Generated in Cardiac Atrial Myocytes Lacking T-Tubules—Insights from a Simulation Study</a></li><li><a href=/readlist/2020/10/lebrecht-2003/>Lebrecht 2003 : Time-dependent and tissue-specific accumulation of mtDNA and respiratory chain defects in chronic doxorubicin cardiomyopathy</a></li><li><a href=/readlist/2020/10/lancaster-2016/>Lancaster 2016 : Improved Prediction of Drug‐Induced Torsades de Pointes Through Simulations of Dynamics and Machine Learning Algorithms</a></li><li><a href=/readlist/2020/10/kussmaul-2006/>Kussmaul 2006 : The mechanism of superoxide production by NADH:ubiquinone oxidoreductase (complex I) from bovine heart mitochondria</a></li><li><a href=/readlist/2020/10/krab-2011/>Krab 2011 : Explaining the enigmatic KMfor oxygen in cytochrome c oxidase: A kinetic model</a></li><li><a href=/readlist/2020/10/kembro-2014/>Kembro 2014 : Complex oscillatory redox dynamics with signaling potential at the edge between normal and pathological mitochondrial function</a></li><li><a href=/readlist/2020/10/kang-2004/>Kang 2004 : Multiple transport modes of the cardiac Na+/Ca2+ exchanger</a></li><li><a href=/readlist/2020/10/jones-2015/>Jones 2015 : Quantification of the effects of electrical remodeling due to hypertrophic cardiomyopathy on human ventricular electromechanical activity and energetics</a></li><li><a href=/readlist/2020/10/jastroch-2010/>Jastroch 2010 : Mitochondrial proton and electron leaks</a></li><li><a href=/readlist/2020/10/jafri-2014/>Jafri 2014 : Modeling Mitochondrial Function and Its Role in Disease</a></li><li><a href=/readlist/2020/10/imperial-2014/>Imperial 2014 : Enzyme kinetic equations of irreversible and reversible reactions in metabolism</a></li><li><a href=/readlist/2020/10/igamberdiev-2015/>Igamberdiev 2015 : Optimization of ATP synthase function in mitochondria and chloroplasts via the adenylate kinase equilibrium</a></li><li><a href=/readlist/2020/10/hettling-2011/>Hettling 2011 : Analyzing the functional properties of the creatine kinase system with multiscale sloppy modeling</a></li><li><a href=/readlist/2020/10/henriksen-2018/>Henriksen 2018 : Anthracycline cardiotoxicity: an update on mechanisms, monitoring and prevention</a></li><li><a href=/readlist/2020/10/heiske-2017/>Heiske 2017 : Comprehensive mathematical model of oxidative phosphorylation valid for physiological and pathological conditions</a></li><li><a href=/readlist/2020/10/heijman-2016/>Heijman 2016 : Computational models of atrial cellular electrophysiology and calcium handling, and their role in atrial fibrillation</a></li><li><a href=/readlist/2020/10/hatano-2015/>Hatano 2015 : Distinct Functional Roles of Cardiac Mitochondrial Subpopulations Revealed by a 3D Simulation Model</a></li><li><a href=/readlist/2020/10/guillaud-2014/>Guillaud 2014 : Superoxide production by cytochrome bc1 complex: a mathematical model</a></li><li><a href=/readlist/2020/10/greenstein-2014/>Greenstein 2014 : Modeling CaMKII-mediated regulation of L-type Ca(2+) channels and ryanodine receptors in the heart</a></li><li><a href=/readlist/2020/10/grandi-2016/>Grandi 2016 : Anti-arrhythmic strategies for atrial fibrillation: The role of computational modeling in discovery, development, and optimization</a></li><li><a href=/readlist/2020/10/gauthier-2013/>Gauthier 2013 : An integrated mitochondrial ROS production and scavenging model: implications for heart failure</a></li><li><a href=/readlist/2020/10/gauthier-2012/>Gauthier 2012 : Toward an integrative computational model of the Guinea pig cardiac myocyte</a></li><li><a href=/readlist/2020/10/garcia-2016/>Garcia 2016 : The ATPase Inhibitory Factor 1 (IF1): A master regulator of energy metabolism and of cell survival</a></li><li><a href=/readlist/2020/10/foteinou-2015/>Foteinou 2015 : Mechanistic investigation of the arrhythmogenic role of oxidized camkii in the heart</a></li><li><a href=/readlist/2020/10/fernandezchas-2018/>Fernandezchas 2018 : Mechanism of doxorubicin cardiotoxicity evaluated by integrating multiple molecular effects into a biophysical model</a></li><li><span class=active>Felmlee 2013 : Mechanism-based pharmacodynamic modeling</span></li><li><a href=/readlist/2020/10/ellinwood-2017/>Ellinwood 2017 : In Silico Assessment of Efficacy and Safety of IKur Inhibitors in Chronic Atrial Fibrillation: Role of Kinetics and State-Dependence of Drug Binding</a></li><li><a href=/readlist/2020/10/deoliveira-2016/>Deoliveira 2016 : A biophysical systems approach to identifying the pathways of acute and chronic doxorubicin mitochondrial cardiotoxicity</a></li><li><a href=/readlist/2020/10/demin-2001/>Demin 2001 : Kinetic Modeling of Energy Metabolism and Superoxide Generation in Hepatocyte Mitochondria</a></li><li><a href=/readlist/2020/10/cortassa-2012/>Cortassa 2012 : Computational modeling of mitochondrial function</a></li><li><a href=/readlist/2020/10/colman-2018/>Colman 2018 : Description of the Human Atrial Action Potential Derived From a Single, Congruent Data Source: Novel Computational Models for Integrated Experimental-Numerical Study of Atrial Arrhythmia Mechanisms</a></li><li><a href=/readlist/2020/10/christensen-2009/>Christensen 2009 : Oxidized calmodulin kinase II regulates conduction following myocardial infarction: a computational analysis</a></li><li><a href=/readlist/2020/10/chinopoulos-2011/>Chinopoulos 2011 : Mitochondrial consumption of cytosolic ATP: Not so fast</a></li><li><a href=/readlist/2020/10/chinopoulos-2010/>Chinopoulos 2010 : Forward operation of adenine nucleotide translocase during F0F1-ATPase reversal: critical role of matrix substrate-level phosphorylation</a></li><li><a href=/readlist/2020/10/chiba-2008/>Chiba 2008 : A simulation study on the activation of cardiac CaMKII delta-isoform and its regulation by phosphatases</a></li><li><a href=/readlist/2020/10/bazil-2016/>Bazil 2016 : Catalytic Coupling of Oxidative Phosphorylation, ATP Demand, and Reactive Oxygen Species Generation</a></li><li><a href=/readlist/2020/10/bazil-2014/>Bazil 2014 : Determining the origins of superoxide and hydrogen peroxide in the mammalian NADH:ubiquinone oxidoreductase</a></li><li><a href=/readlist/2020/10/bazil-2013/>Bazil 2013: Analysis of the Kinetics and Bistability of Ubiquinol:Cytochrome c Oxidoreductase</a></li><li><a href=/readlist/2020/10/bazil-2010/>Bazil 2010 : Modeling mitochondrial bioenergetics with integrated volume dynamics</a></li><li><a href=/readlist/2020/10/barros-2012/>Barros 2012: Simulations of Complex and Microscopic Models of Cardiac Electrophysiology Powered by Multi-GPU Platforms</a></li></ul></nav></div></div><div class="details toc" id=toc-static kept><div class="details-summary toc-title"><span>Contents</span>
<span><i class="details-icon fas fa-angle-right"></i></span></div><div class="details-content toc-content" id=toc-content-static><nav id=TableOfContents><ul><li><a href=#introduction>Introduction</a></li><li><a href=#modeling-requirements>Modeling Requirements</a><ul><li><a href=#simple-direct-effect-models>Simple Direct Effect Models</a></li><li><a href=#biophase-distribution>Biophase Distribution</a></li><li><a href=#indirect-response-models>Indirect Response Models</a><ul><li><a href=#model-1>Model 1</a></li><li><a href=#model-2>Model 2</a></li><li><a href=#model-3>Model 3</a></li><li><a href=#model-4>Model 4</a></li></ul></li><li><a href=#signal-transduction-models>Signal Transduction Models</a></li><li><a href=#irreversible-effect-models>Irreversible Effect Models</a></li><li><a href=#more-complex-models>More Complex Models</a><ul><li><a href=#the-time-course-of-paraoxon-inactivation-of-in-vitro-whole-blood-cholinesterase-wbche>The time-course of paraoxon inactivation of in vitro whole blood cholinesterase (WBChE)</a></li></ul></li></ul></li><li><a href=#prospects>Prospects</a></li></ul></nav></div></div><div class=content id=content><p><a href=https://sciwheel.com/work/#/items/5848333 target=_blank rel="noopener noreffer">Sciwheel</a><sup id=fnref:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup>, <a href=http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160 target=_blank rel="noopener noreffer">PMC3684160</a></p><h2 id=introduction class=headerLink><a href=#introduction class=header-mark></a>Introduction</h2><ul><li>diseases and both types of drug responses may emerge from perturbations of singular complex interconnected networks</li></ul><h2 id=modeling-requirements class=headerLink><a href=#modeling-requirements class=header-mark></a>Modeling Requirements</h2><ul><li>Useful pharmacodynamic models are based on plausible mathematical and pharmacological exposure–response relationships<figure><a class=lightgallery href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f1.jpg title=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f1.jpg data-thumbnail=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f1.jpg data-sub-html="<h2>Basic components of pharmacodynamic models</h2>"><img class=lazyload data-src=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f1.jpg data-srcset="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f1.jpg, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f1.jpg 1.5x, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f1.jpg 2x" data-sizes=auto alt=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f1.jpg></a><figcaption class=image-caption>Basic components of pharmacodynamic models</figcaption></figure></li><li>relevant biological fluid (e.g., plasma, Cp) or the biophase (Ce)</li><li></li><li>A good graphical analysis (along with a priori knowledge of drug mechanisms) may be used to narrow down the number of structural models</li><li>good initial parameter estimates can reduce the likelihood of falling into local minima</li><li>fitting a model to concentration–time profiles in relevant biological fluids</li><li>Although simultaneous PK/PD modeling is desirable, this can still be a formidable challenge for complex models</li></ul><h3 id=simple-direct-effect-models class=headerLink><a href=#simple-direct-effect-models class=header-mark></a>Simple Direct Effect Models</h3><ul><li>Emax model: The full Hill equation, or sigmoid Emax model, incorporates a curve-fitting parameter, γ, which describes the steepness of the concentration–effect relationship</li></ul><p>$$
E=E_{0} \pm \frac{E_{m a x} \times C_{\mathrm{p}}^{\gamma}}{\mathrm{EC}<em>{50}^{\gamma}+C</em>{\mathrm{p}}^{\gamma}}
$$</p><p><img class=lazyload data-src=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f2.jpg data-srcset="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f2.jpg, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f2.jpg 1.5x, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f2.jpg 2x" data-sizes=auto alt=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f2.jpg title=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f2.jpg></p><ul><li>linear slope of the effect:
$$
m=\frac{E_{\max } \times \gamma}{4}
$$</li></ul><p><figure><a class=lightgallery href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f3.jpg title=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f3.jpg data-thumbnail=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f3.jpg data-sub-html="<h2>Direct effect model of tacrolimus-induced changes of QTc intervals in guinea pigs</h2>"><img class=lazyload data-src=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f3.jpg data-srcset="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f3.jpg, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f3.jpg 1.5x, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f3.jpg 2x" data-sizes=auto alt=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f3.jpg></a><figcaption class=image-caption>Direct effect model of tacrolimus-induced changes of QTc intervals in guinea pigs</figcaption></figure></p><h3 id=biophase-distribution class=headerLink><a href=#biophase-distribution class=header-mark></a>Biophase Distribution</h3><ul><li><p>in vivo pharmacological effects will lag behind plasma drug concentrations: hysteresis, or a temporal disconnect in effect versus concentration plots</p></li><li><p>drug effect through a hypothetical effect compartment<figure><a class=lightgallery href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f4.jpg title=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f4.jpg data-thumbnail=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f4.jpg data-sub-html="<h2>Biophase model structure</h2>"><img class=lazyload data-src=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f4.jpg data-srcset="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f4.jpg, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f4.jpg 1.5x, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f4.jpg 2x" data-sizes=auto alt=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f4.jpg></a><figcaption class=image-caption>Biophase model structure</figcaption></figure></p></li><li><p>The biophase model is only suitable for describing delayed responses due to drug distribution</p></li></ul><h3 id=indirect-response-models class=headerLink><a href=#indirect-response-models class=header-mark></a>Indirect Response Models</h3><ul><li>The four basic models include inhibition of production (Model I) or dissipation (Model II) of response or stimulation of production (Model III) or dissipation of response (Model IV)</li></ul><p><img class=lazyload data-src=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f5.jpg data-srcset="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f5.jpg, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f5.jpg 1.5x, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f5.jpg 2x" data-sizes=auto alt=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f5.jpg title=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f5.jpg></p><h4 id=model-1 class=headerLink><a href=#model-1 class=header-mark></a>Model 1</h4><p>$$
\frac{\mathrm{d} R}{\mathrm{d} t}=k_{\mathrm{in}}\left(1-\frac{I_{\mathrm{max}} \times C_{\mathrm{p}}}{\mathrm{IC}<em>{50}+C</em>{\mathrm{p}}}\right)-k_{\mathrm{out}} \times R
$$</p><h4 id=model-2 class=headerLink><a href=#model-2 class=header-mark></a>Model 2</h4><p>$$
\frac{\mathrm{d} R}{\mathrm{d} t}=k_{\mathrm{in}}-k_{\mathrm{out}}\left(1-\frac{I_{\mathrm{max}} \times C_{\mathrm{p}}}{\mathrm{IC}<em>{50}+C</em>{\mathrm{p}}}\right) R
$$</p><h4 id=model-3 class=headerLink><a href=#model-3 class=header-mark></a>Model 3</h4><p>$$
\frac{\mathrm{d} R}{\mathrm{d} t}=k_{\mathrm{in}}\left(1+\frac{S_{m a x} \times C_{\mathrm{p}}}{\mathrm{SC}<em>{50}+C</em>{\mathrm{p}}}\right)-k_{\mathrm{out}} \times R
$$</p><h4 id=model-4 class=headerLink><a href=#model-4 class=header-mark></a>Model 4</h4><p>$$
\frac{\mathrm{d} R}{\mathrm{d} t}=k_{\mathrm{in}}-k_{\mathrm{out}}\left(1+\frac{S_{\mathrm{max}} \times C_{\mathrm{p}}}{\mathrm{SC}<em>{50}+C</em>{\mathrm{p}}}\right) R
$$</p><ul><li>The basic indirect response models can be extended to incorporate a precursor compartment (P)
<img class=lazyload data-src=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f6.jpg data-srcset="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f6.jpg, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f6.jpg 1.5x, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f6.jpg 2x" data-sizes=auto alt=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f6.jpg title=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f6.jpg></li></ul><p><img class=lazyload data-src=https://user-images.githubusercontent.com/40054455/96898411-40e6a380-14c2-11eb-9b17-be0d12f169e4.png data-srcset="https://user-images.githubusercontent.com/40054455/96898411-40e6a380-14c2-11eb-9b17-be0d12f169e4.png, https://user-images.githubusercontent.com/40054455/96898411-40e6a380-14c2-11eb-9b17-be0d12f169e4.png 1.5x, https://user-images.githubusercontent.com/40054455/96898411-40e6a380-14c2-11eb-9b17-be0d12f169e4.png 2x" data-sizes=auto alt=https://user-images.githubusercontent.com/40054455/96898411-40e6a380-14c2-11eb-9b17-be0d12f169e4.png title=image></p><h3 id=signal-transduction-models class=headerLink><a href=#signal-transduction-models class=header-mark></a>Signal Transduction Models</h3><ul><li>a lag between drug concentration and observed effects owing to time-dependent signal transduction: <strong>delayed differential equations</strong> (DDEs)?</li><li>rate of initial transit compartment (M1)
$$
\frac{\mathrm{d} M_{1}}{\mathrm{d} t}=\frac{1}{\tau}\left(\frac{E_{m a x} \times C_{\mathrm{p}}}{\mathrm{EC}<em>{50}+C</em>{\mathrm{p}}}-M_{1}\right)
$$
For the ith compartment:
$$
\frac{\mathrm{d} M_{i}}{\mathrm{d} t}=\frac{1}{\tau}\left(M_{i-1}-M_{i}\right)
$$</li><li>e.g. Chemotherapy-induced myelosuppression
<img class=lazyload data-src=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f7.jpg data-srcset="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f7.jpg, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f7.jpg 1.5x, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f7.jpg 2x" data-sizes=auto alt=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f7.jpg title=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f7.jpg></li></ul><h3 id=irreversible-effect-models class=headerLink><a href=#irreversible-effect-models class=header-mark></a>Irreversible Effect Models</h3><p>$$
\frac{\mathrm{d} R}{\mathrm{d} t}=-k \times C \times R
$$</p><ul><li>This approach is only applicable for non-proliferating cell populations, but may be extended to incorporate cell growth
$$
\frac{\mathrm{d} R}{\mathrm{d} t}=k_{s} \times R-k \times C \times R
$$
<img class=lazyload data-src=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f8.jpg data-srcset="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f8.jpg, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f8.jpg 1.5x, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f8.jpg 2x" data-sizes=auto alt=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f8.jpg title=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684160/bin/nihms472103f8.jpg></li><li>The irreversible effect model can also be adapted to include the turnover or production and loss of a biomarker
$$
\frac{\mathrm{d} R}{\mathrm{d} t}=k_{\mathrm{in}}-k_{\mathrm{out}} R-k \times C \times R
$$</li></ul><h3 id=more-complex-models class=headerLink><a href=#more-complex-models class=header-mark></a>More Complex Models</h3><h4 id=the-time-course-of-paraoxon-inactivation-of-in-vitro-whole-blood-cholinesterase-wbche class=headerLink><a href=#the-time-course-of-paraoxon-inactivation-of-in-vitro-whole-blood-cholinesterase-wbche class=header-mark></a>The time-course of paraoxon inactivation of in vitro whole blood cholinesterase (WBChE)</h4><p>$$
\frac{\mathrm{d} E_{\mathrm{A}}}{\mathrm{d} t}=-\left(\frac{k C_{\mathrm{PO}}}{\mathrm{EC}<em>{50, \mathrm{PO}}+C</em>{\mathrm{PO}}}\right) E_{\mathrm{A}}+k_{r} E_{\mathrm{I}}
$$
$$
\frac{\mathrm{d} E_{I}}{\mathrm{d} t}=\left(\frac{k C_{\mathrm{PO}}}{\mathrm{EC}<em>{50, \mathrm{PO}}+C</em>{\mathrm{PO}}}\right) E_{A}-\left(k_{r}+k_{\mathrm{age}}\right) E_{1}
$$</p><ul><li>The reactivation of this in vitro system by PRX was modeled as an indirect response
$$
\frac{\mathrm{d} E_{\mathrm{A}}}{\mathrm{d} t}=-\left(\frac{k C_{\mathrm{PO}}}{\mathrm{EC}<em>{50, \mathrm{PO}}+C</em>{\mathrm{PO}}}\right) E_{\mathrm{A}}+k_{\mathrm{r}}\left(1+\frac{E_{\mathrm{max}} C_{\mathrm{PRX}}^{h}}{\mathrm{EC}<em>{50, \mathrm{PRX}}^{h}+C</em>{\mathrm{PRX}}^{h}}\right) E_{\mathrm{I}}
$$
$$
\begin{aligned} \frac{\mathrm{d} E_{\mathrm{I}}}{\mathrm{d} t}=\left(\frac{k C_{\mathrm{PO}}}{\mathrm{EC}<em>{50, \mathrm{PO}}+C</em>{\mathrm{PO}}}\right) & E_{\mathrm{A}}-k_{\mathrm{r}}\left(1+\frac{E_{\mathrm{max}} C_{\mathrm{PRX}}^{h}}{\mathrm{EC}<em>{50, \mathrm{PRX}}^{h}+C</em>{\mathrm{PRX}}^{h}}\right) E_{\mathrm{I}} \cr &-\left(k_{\mathrm{age}} E_{\mathrm{I}}\right) \end{aligned}
$$</li><li>the toxicodynamic biomarker, expiratory time (TE), was linked to apparent active enzyme (EA) according to the following nonlinear transfer function
$$
T_{\mathrm{E}}=T_{\mathrm{E}}^{0}+\frac{E_{\mathrm{max}, T_{\mathrm{E}}}\left(\frac{E_{0}}{E_{\mathrm{A}}}-1\right)^{n}}{E_{50}^{n}+\left(\frac{E_{0}}{E_{\mathrm{A}}}-1\right)^{n}}
$$</li></ul><h2 id=prospects class=headerLink><a href=#prospects class=header-mark></a>Prospects</h2><ul><li>The future of mechanism-based pharmacodynamic modeling for both therapeutic and adverse drug responses is promising for model-based drug development and therapeutics</li><li>A diverse array of models is available with a minimal number of identifiable parameters to mimic mechanisms and the time-course of therapeutic and adverse drug effects</li><li>Network-based systems pharmacology models have shown utility for understanding drug-induced adverse events (e.g. FBA, FVA)</li></ul><div class=footnotes role=doc-endnotes><hr><ol><li id=fn:1><p>Felmlee MA, Morris ME, Mager DE. Mechanism-based pharmacodynamic modeling. Methods Mol Biol. 2012;929:583–600.&#160;<a href=#fnref:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li></ol></div></div><div class=post-footer id=post-footer><div class=post-info><div class=post-info-line><div class=post-info-mod><span>Updated on 2022-07-07&nbsp;<a class=git-hash href=https://github.com/sosiristseng/readlist/commit/640e6964e343f4b8efe341b974d3ff1abf09f281 target=_blank title="commit by Wen-Wei Tseng(sosiristseng@gmail.com) 640e6964e343f4b8efe341b974d3ff1abf09f281: move files" rel="noopener noreferrer">
<i class="fas fa-hashtag fa-fw"></i>640e696</a></span></div><div class=post-info-license></div></div><div class=post-info-line><div class=post-info-md><span><a class=link-to-mardown href=/readlist/2020/10/felmlee-2013/index.md target=_blank rel="noopener noreferrer">Read markdown</a>
</span><span>|&nbsp;<a class=link-to-source href=https://github.com/sosiristseng/readlist/blob/main/content/posts/f/felmlee-2013.md target=_blank rel="noopener noreferrer">View source</a>
</span><span>|&nbsp;<a class=link-to-edit href=https://github.com/sosiristseng/readlist/edit/main/content/posts/f/felmlee-2013.md target=_blank rel="noopener noreferrer">Edit this page</a></span></div><div class=post-info-share><span><a href=# onclick=return!1 title="Share on Twitter" data-sharer=twitter data-url=https://sosiristseng.github.io/readlist/2020/10/felmlee-2013/ data-title="Felmlee 2013 : Mechanism-based pharmacodynamic modeling" data-via=sosiristseng data-hashtags="quantitative systems pharmacology,enzyme kinetics"><i class="fab fa-twitter fa-fw"></i></a><a href=# onclick=return!1 title="Share on Facebook" data-sharer=facebook data-url=https://sosiristseng.github.io/readlist/2020/10/felmlee-2013/ data-hashtag="quantitative systems pharmacology"><i class="fab fa-facebook-square fa-fw"></i></a><a href=# onclick=return!1 title="Share on Hacker News" data-sharer=hackernews data-url=https://sosiristseng.github.io/readlist/2020/10/felmlee-2013/ data-title="Felmlee 2013 : Mechanism-based pharmacodynamic modeling"><i class="fab fa-hacker-news fa-fw"></i></a><a href=# onclick=return!1 title="Share on Reddit" data-sharer=reddit data-url=https://sosiristseng.github.io/readlist/2020/10/felmlee-2013/><i class="fab fa-reddit fa-fw"></i></a><a href=# onclick=return!1 title="Share on Line" data-sharer=line data-url=https://sosiristseng.github.io/readlist/2020/10/felmlee-2013/ data-title="Felmlee 2013 : Mechanism-based pharmacodynamic modeling"><i data-svg-src=/readlist/lib/simple-icons/icons/line.min.svg></i></a><a href=# onclick=return!1 title="Share on Telegram" data-sharer=telegram data-url=https://sosiristseng.github.io/readlist/2020/10/felmlee-2013/ data-title="Felmlee 2013 : Mechanism-based pharmacodynamic modeling" data-web><i class="fab fa-telegram-plane fa-fw"></i></a></span></div></div></div><div class=post-info-more><section class=post-tags><i class="fas fa-tags fa-fw"></i>&nbsp;<a href=/readlist/tags/quantitative-systems-pharmacology/>quantitative systems pharmacology</a>,&nbsp;<a href=/readlist/tags/enzyme-kinetics/>enzyme kinetics</a></section><section><span><a href=javascript:void(0); onclick=window.history.back()>Back</a></span>&nbsp;|&nbsp;<span><a href=/readlist/>Home</a></span></section></div><div class=post-nav><a href=/readlist/2020/10/ellinwood-2017/ class=prev rel=prev title="Ellinwood 2017 : In Silico Assessment of Efficacy and Safety of IKur Inhibitors in Chronic Atrial Fibrillation: Role of Kinetics and State-Dependence of Drug Binding"><i class="fas fa-angle-left fa-fw"></i>Ellinwood 2017 : In Silico Assessment of Efficacy and Safety of IKur Inhibitors in Chronic Atrial Fibrillation: Role of Kinetics and State-Dependence of Drug Binding</a>
<a href=/readlist/2020/10/fernandezchas-2018/ class=next rel=next title="Fernandezchas 2018 : Mechanism of doxorubicin cardiotoxicity evaluated by integrating multiple molecular effects into a biophysical model">Fernandezchas 2018 : Mechanism of doxorubicin cardiotoxicity evaluated by integrating multiple molecular effects into a biophysical model<i class="fas fa-angle-right fa-fw"></i></a></div></div></article></div></main><footer class=footer><div class=footer-container><div class=footer-line>Powered by <a href=https://gohugo.io/ target=_blank rel="noopener noreffer" title="Hugo 0.101.0">Hugo</a>&nbsp;|&nbsp;Hosted on <a title="Github Pages" href=https://docs.github.com/en/pages/ target=_blank rel="noopener noreffer">GitHub Pages</a>&nbsp;|&nbsp;Theme - <a href=https://github.com/HEIGE-PCloud/DoIt target=_blank rel="noopener noreffer" title="DoIt 0.2.13"><i class="far fa-edit fa-fw"></i> DoIt</a></div><div class=footer-line><i class="far fa-copyright fa-fw"></i><span itemprop=copyrightYear>2022</span><span class=author itemprop=copyrightHolder>&nbsp;<a href=https://github.com/sosiristseng target=_blank rel="noopener noreferrer">Wen-Wei Tseng</a></span>&nbsp;|&nbsp;<span class=license><a rel="license external nofollow noopener noreferrer" href=https://creativecommons.org/licenses/by-nc/4.0/ target=_blank>CC BY-NC 4.0</a></span></div><div class=footer-line></div><div class=footer-line></div></div><script>"serviceWorker"in navigator&&(navigator.serviceWorker.register("/sw.min.js",{scope:"/"}).then(function(){}),navigator.serviceWorker.ready.then(function(){}))</script></footer></div><div id=fixed-buttons><a href=#back-to-top id=back-to-top-button class=fixed-button title="Back to Top"><i class="fas fa-arrow-up fa-fw"></i>
</a><a href=# id=view-comments class=fixed-button title="View Comments"><i class="fas fa-comment fa-fw"></i></a></div><div class=assets><script type=text/javascript src=/readlist/lib/autocomplete/autocomplete.min.js></script><script type=text/javascript src=/readlist/lib/fuse/fuse.min.js></script><script type=text/javascript src=/readlist/lib/lazysizes/lazysizes.min.js></script><script type=text/javascript src=/readlist/lib/topbar/topbar.min.js></script><script type=text/javascript src=/readlist/lib/pjax/pjax.min.js></script><script type=text/javascript src=/readlist/js/theme.min.js defer></script></div><div class=pjax-assets><script type=text/javascript>window.config={code:{copyTitle:"Copy to clipboard",maxShownLines:15},comment:{},lightGallery:{actualSize:!1,exThumbImage:"data-thumbnail",hideBarsDelay:2e3,selector:".lightgallery",speed:400,thumbContHeight:80,thumbWidth:80,thumbnail:!0},math:{delimiters:[{display:!0,left:"$$",right:"$$"},{display:!0,left:"\\[",right:"\\]"},{display:!1,left:"$",right:"$"},{display:!1,left:"\\(",right:"\\)"}],strict:!1},search:{distance:100,findAllMatches:!1,fuseIndexURL:"/readlist/index.json",highlightTag:"em",ignoreFieldNorm:!1,ignoreLocation:!1,isCaseSensitive:!1,location:0,maxResultLength:10,minMatchCharLength:3,noResultsFound:"No results found",snippetLength:30,threshold:.3,type:"fuse",useExtendedSearch:!1},sharerjs:!0}</script><script type=text/javascript src=/readlist/lib/lightgallery/lightgallery.min.js></script><script type=text/javascript src=/readlist/lib/lightgallery/lg-thumbnail.min.js></script><script type=text/javascript src=/readlist/lib/lightgallery/lg-zoom.min.js></script><script type=text/javascript src=/readlist/lib/clipboard/clipboard.min.js></script><script type=text/javascript src=/readlist/lib/sharer/sharer.min.js></script><script type=text/javascript src=/readlist/lib/katex/katex.min.js defer></script><script type=text/javascript src=/readlist/lib/katex/auto-render.min.js defer></script><script type=text/javascript src=/readlist/lib/katex/copy-tex.min.js defer></script><script type=text/javascript src=/readlist/lib/katex/mhchem.min.js defer></script><script type=text/javascript src=/readlist/js/katex.min.js defer></script><link rel=stylesheet href=/readlist/lib/lightgallery/lightgallery.min.css><link rel=stylesheet href=/readlist/lib/katex/katex.min.css><link rel=preload as=style onload='this.onload=null,this.rel="stylesheet"' href=/readlist/lib/katex/copy-tex.min.css><noscript><link rel=stylesheet href=/readlist/lib/katex/copy-tex.min.css></noscript></div></body></html>