<!doctype html><html lang=en><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta name=robots content="noodp"><title class=pjax-title>Goldberg 2017 : Emerging whole-cell modeling principles and methods - Reading List</title><meta name=Description content="Reading Note of Wen-Wei"><meta property="og:title" content="Goldberg 2017 : Emerging whole-cell modeling principles and methods"><meta property="og:description" content="Sciwheel 1, PMC5997489
Whole-cell (WC) computational models aim to predict cellular phenotypes from genotype and the environment by representing the function of each gene, gene product, and metabolite."><meta property="og:type" content="article"><meta property="og:url" content="https://sosiristseng.github.io/readlist/2021/04/goldberg-2017/"><meta property="og:image" content="https://sosiristseng.github.io/readlist/"><meta property="article:section" content="posts"><meta property="article:published_time" content="2021-04-08T16:53:04+08:00"><meta property="article:modified_time" content="2022-07-07T21:00:38+08:00"><meta property="og:site_name" content="Reading List"><meta name=twitter:card content="summary_large_image"><meta name=twitter:image content="https://sosiristseng.github.io/readlist/"><meta name=twitter:title content="Goldberg 2017 : Emerging whole-cell modeling principles and methods"><meta name=twitter:description content="Sciwheel 1, PMC5997489
Whole-cell (WC) computational models aim to predict cellular phenotypes from genotype and the environment by representing the function of each gene, gene product, and metabolite."><meta name=application-name content="Reading Notes"><meta name=apple-mobile-web-app-title content="Reading Notes"><meta name=theme-color content="#f8f8f8"><meta name=msapplication-TileColor content="#da532c"><link rel="shortcut icon" type=image/x-icon href=/favicon.ico><link rel=icon type=image/png sizes=32x32 href=/favicon-32x32.png><link rel=icon type=image/png sizes=16x16 href=/favicon-16x16.png><link rel=apple-touch-icon sizes=180x180 href=/apple-touch-icon.png><link rel=mask-icon href=/safari-pinned-tab.svg color=#5bbad5><link rel=manifest href=/site.webmanifest><link rel=canonical href=https://sosiristseng.github.io/readlist/2021/04/goldberg-2017/><link rel=prev href=https://sosiristseng.github.io/readlist/2021/01/gregg-2016/><link rel=next href=https://sosiristseng.github.io/readlist/2021/04/zamponi-2018/><link rel=stylesheet href=/readlist/lib/normalize/normalize.min.css><link rel=stylesheet href=/readlist/css/style.min.css><link rel=preload as=style onload='this.onload=null,this.rel="stylesheet"' href=/readlist/lib/fontawesome-free/all.min.css><noscript><link rel=stylesheet href=/readlist/lib/fontawesome-free/all.min.css></noscript><link rel=preload as=style onload='this.onload=null,this.rel="stylesheet"' href=/readlist/lib/animate/animate.min.css><noscript><link rel=stylesheet href=/readlist/lib/animate/animate.min.css></noscript><script type=application/ld+json>{"@context":"http://schema.org","@type":"BlogPosting","headline":"Goldberg 2017 : Emerging whole-cell modeling principles and methods","inLanguage":"en","mainEntityOfPage":{"@type":"WebPage","@id":"https:\/\/sosiristseng.github.io\/readlist\/2021\/04\/goldberg-2017\/"},"genre":"posts","keywords":"ODE","wordcount":522,"url":"https:\/\/sosiristseng.github.io\/readlist\/2021\/04\/goldberg-2017\/","datePublished":"2021-04-08T16:53:04+08:00","dateModified":"2022-07-07T21:00:38+08:00","license":"This work is licensed under a Creative Commons Attribution-NonCommercial 4.0 International License.","publisher":{"@type":"Organization","name":"xxxx"},"author":{"@type":"Person","name":"Goldberg and others"},"description":""}</script></head><body header-desktop=fixed header-mobile=auto><script type=text/javascript>function setTheme(e){document.body.setAttribute("theme",e),document.documentElement.style.setProperty("color-scheme",e==="light"?"light":"dark")}function saveTheme(e){window.localStorage&&localStorage.setItem("theme",e)}function getMeta(e){const t=document.getElementsByTagName("meta");for(let n=0;n<t.length;n++)if(t[n].getAttribute("name")===e)return t[n];return""}if(window.localStorage&&localStorage.getItem("theme")){let e=localStorage.getItem("theme");e==="light"||e==="dark"||e==="black"?setTheme(e):setTheme(window.matchMedia&&window.matchMedia("(prefers-color-scheme: dark)").matches?"dark":"light")}else"auto"==="light"||"auto"==="dark"||"auto"==="black"?(setTheme("auto"),saveTheme("auto")):(saveTheme("auto"),setTheme(window.matchMedia&&window.matchMedia("(prefers-color-scheme: dark)").matches?"dark":"light"));let metaColors={light:"#f8f8f8",dark:"#252627",black:"#000000"};getMeta("theme-color").content=metaColors[document.body.getAttribute("theme")]</script><div id=back-to-top></div><div id=mask></div><div class=wrapper><header class=desktop id=header-desktop><div class=header-wrapper><div class=header-title><a href=/readlist/ title="Reading List"><span class=header-title-pre><i class='fas fa-edit fa-fw'></i></span>Reading List</a></div><div class=menu><div class=menu-inner><a class=menu-item href=/readlist/posts/>All Reading Notes </a><a class=menu-item href=/readlist/tags/>Tags </a><a class=menu-item href=/readlist/categories/>Categories </a><a class=menu-item href=/readlist/series/>Series </a><a class=menu-item href=https://github.com/sosiristseng/readlist title=GitHub rel="noopener noreffer" target=_blank><i class='fab fa-github fa-fw'></i> </a><span class="menu-item delimiter"></span><span class="menu-item search" id=search-desktop>
<input type=text placeholder=Search id=search-input-desktop>
<a href=# onclick=return!1 class="search-button search-toggle" id=search-toggle-desktop title=Search><i class="fas fa-search fa-fw"></i></a>
<a href=# onclick=return!1 class="search-button search-clear" id=search-clear-desktop title=Clear><i class="fas fa-times-circle fa-fw"></i></a>
<span class="search-button search-loading" id=search-loading-desktop><i class="fas fa-spinner fa-fw fa-spin"></i></span>
</span><a href=# onclick=return!1 class="menu-item theme-switch" title="Switch Theme"><i class="fas fa-adjust fa-fw"></i></a></div></div></div></header><header class=mobile id=header-mobile><div class=header-container><div class=header-wrapper><div class=header-title><a href=/readlist/ title="Reading List"><span class=header-title-pre><i class='fas fa-edit fa-fw'></i></span>Reading List</a></div><div class=menu-toggle id=menu-toggle-mobile><span></span><span></span><span></span></div></div><div class=menu id=menu-mobile><div class=search-wrapper><div class="search mobile" id=search-mobile><input type=text placeholder=Search id=search-input-mobile>
<a href=# onclick=return!1 class="search-button search-toggle" id=search-toggle-mobile title=Search><i class="fas fa-search fa-fw"></i></a>
<a href=# onclick=return!1 class="search-button search-clear" id=search-clear-mobile title=Clear><i class="fas fa-times-circle fa-fw"></i></a>
<span class="search-button search-loading" id=search-loading-mobile><i class="fas fa-spinner fa-fw fa-spin"></i></span></div><a href=# onclick=return!1 class=search-cancel id=search-cancel-mobile>Cancel</a></div><a class=menu-item href=/readlist/posts/ title>All Reading Notes</a><a class=menu-item href=/readlist/tags/ title>Tags</a><a class=menu-item href=/readlist/categories/ title>Categories</a><a class=menu-item href=/readlist/series/ title>Series</a><a class=menu-item href=https://github.com/sosiristseng/readlist title=GitHub rel="noopener noreffer" target=_blank><i class='fab fa-github fa-fw'></i></a><a href=# onclick=return!1 class="menu-item theme-switch" title="Switch Theme">
<i class="fas fa-adjust fa-fw"></i></a></div></div></header><div class="search-dropdown desktop"><div id=search-dropdown-desktop></div></div><div class="search-dropdown mobile"><div id=search-dropdown-mobile></div></div><main class=main><div class=container><div class=toc id=toc-auto><h2 class=toc-title>Contents</h2><div class=toc-content id=toc-content-auto><nav id=TableOfContents><ul><li><a href=#available-resources>AVAILABLE RESOURCES</a><ul><li><a href=#measurement-methods>Measurement methods</a></li><li><a href=#data-repositories>Data repositories</a></li><li><a href=#prediction-tools>Prediction tools</a></li><li><a href=#published-models>Published models</a></li><li><a href=#emerging-methods-and-tools-data-aggregation-and-organization>Emerging methods and tools, Data aggregation and organization</a></li><li><a href=#scalable-model-design>Scalable model design</a></li><li><a href=#model-languages>Model languages</a></li><li><a href=#simulation-tools>Simulation tools</a></li><li><a href=#calibration>Calibration</a></li><li><a href=#verification>Verification</a></li><li><a href=#simulation-results-analysis>Simulation results analysis</a></li></ul></li><li><a href=#technological-challenges>TECHNOLOGICAL CHALLENGES</a></li><li><a href=#highlights>Highlights</a></li></ul></nav></div></div><script>document.getElementsByTagName("main")[0].setAttribute("pageStyle","normal")</script><script>document.getElementsByTagName("main")[0].setAttribute("autoTOC","true")</script><article class="page single"><h1 class="single-title animate__animated animate__flipInX">Goldberg 2017 : Emerging whole-cell modeling principles and methods</h1><div class=post-meta><div class=post-meta-line><span class=post-author><i class="author fas fa-user-circle fa-fw"></i><a href=https://github.com/sosiristseng title=Author target=_blank rel="noopener noreffer author" class=author>Goldberg and others</a>
</span>&nbsp;<span class=post-category>included in </span>&nbsp;<span class=post-category>category <a href=/readlist/categories/review/><i class="far fa-folder fa-fw"></i>Review</a></span></div><div class=post-meta-line><i class="far fa-calendar-alt fa-fw"></i>&nbsp;<time datetime=2021-04-08>2021-04-08</time>&nbsp;<i class="far fa-edit fa-fw"></i>&nbsp;<time datetime=2022-07-07>2022-07-07</time>&nbsp;<i class="fas fa-pencil-alt fa-fw"></i>&nbsp;522 words&nbsp;
<i class="far fa-clock fa-fw"></i>&nbsp;3 minutes&nbsp;</div></div><div class="details toc" id=toc-static kept><div class="details-summary toc-title"><span>Contents</span>
<span><i class="details-icon fas fa-angle-right"></i></span></div><div class="details-content toc-content" id=toc-content-static><nav id=TableOfContents><ul><li><a href=#available-resources>AVAILABLE RESOURCES</a><ul><li><a href=#measurement-methods>Measurement methods</a></li><li><a href=#data-repositories>Data repositories</a></li><li><a href=#prediction-tools>Prediction tools</a></li><li><a href=#published-models>Published models</a></li><li><a href=#emerging-methods-and-tools-data-aggregation-and-organization>Emerging methods and tools, Data aggregation and organization</a></li><li><a href=#scalable-model-design>Scalable model design</a></li><li><a href=#model-languages>Model languages</a></li><li><a href=#simulation-tools>Simulation tools</a></li><li><a href=#calibration>Calibration</a></li><li><a href=#verification>Verification</a></li><li><a href=#simulation-results-analysis>Simulation results analysis</a></li></ul></li><li><a href=#technological-challenges>TECHNOLOGICAL CHALLENGES</a></li><li><a href=#highlights>Highlights</a></li></ul></nav></div></div><div class=content id=content><p><a href=https://sciwheel.com/work/#/items/4651953 target=_blank rel="noopener noreffer">Sciwheel</a> <sup id=fnref:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup>, <a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5997489/ target=_blank rel="noopener noreffer">PMC5997489</a></p><p>Whole-cell (WC) computational models aim to predict cellular phenotypes from genotype and the environment by representing the function of each gene, gene product, and metabolite.</p><p>We propose that WC models aim to represent all of the chemical reactions in a cell and all of the physical processes that influence their rates</p><ul><li>the <strong>sequence</strong> of each chromosome, RNA, and protein</li><li>the <strong>structure</strong> of each molecule</li><li>the <strong>subcellular organization</strong> of cells into organelles and microdomains</li><li>the participants and effect of each <strong>molecular interaction</strong></li><li>the <strong>kinetic parameters</strong> of each interaction</li><li>the <strong>concentration</strong> of each species in each organelle and microdomain</li><li>the concentration of each species in the <strong>extracellular environment</strong></li></ul><p>The things WC models aim to proedict:</p><ul><li><strong>stochastic dynamics</strong> of each molecular interaction</li><li><strong>temporal dynamics</strong> of the concentration of each species</li><li><strong>spatial dynamics</strong> of the concentration of each species in each organelle and microdomain</li><li>complex <strong>phenotypes</strong> such as cell shape, growth rate, motility, and fate, as well as the <strong>variation</strong> in the behavior of single cells within clonal populations</li></ul><p>enables WC models to generate predictions that could be embedded into higher-order multiscale models.</p><p><img class=lazyload data-src=https://user-images.githubusercontent.com/40054455/113998198-69553500-988b-11eb-8512-b84feb08c3db.png data-srcset="https://user-images.githubusercontent.com/40054455/113998198-69553500-988b-11eb-8512-b84feb08c3db.png, https://user-images.githubusercontent.com/40054455/113998198-69553500-988b-11eb-8512-b84feb08c3db.png 1.5x, https://user-images.githubusercontent.com/40054455/113998198-69553500-988b-11eb-8512-b84feb08c3db.png 2x" data-sizes=auto alt=https://user-images.githubusercontent.com/40054455/113998198-69553500-988b-11eb-8512-b84feb08c3db.png title="Fig 1"></p><h2 id=available-resources class=headerLink><a href=#available-resources class=header-mark></a>AVAILABLE RESOURCES</h2><p>WC models require a ton of parameters.</p><h3 id=measurement-methods class=headerLink><a href=#measurement-methods class=header-mark></a>Measurement methods</h3><p>single-cell and genomic measurement</p><ul><li>Meth-Seq (<a href=https://pubmed.ncbi.nlm.nih.gov/20125086/ target=_blank rel="noopener noreffer">genomewide DNA methylation analysis</a>.)</li><li>Hi-C (<a href=https://pubmed.ncbi.nlm.nih.gov/23657480/ target=_blank rel="noopener noreffer">chromosome structures</a>)</li><li>ChIP-seq (<a href=https://pubmed.ncbi.nlm.nih.gov/19736561/ target=_blank rel="noopener noreffer">protein-DNA interactions</a>)</li><li>fluorescence microscopy: protein localizations</li><li>mass-spectrometry: metabolite and protein concentrations</li><li>FISH and scRNA-seq: dynamics and single-cell variation of RNA abundances</li></ul><h3 id=data-repositories class=headerLink><a href=#data-repositories class=header-mark></a>Data repositories</h3><p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5997489/bin/NIHMS928198-supplement.xlsx target=_blank rel="noopener noreffer">A list of public repositories</a> (in Excel file).</p><h3 id=prediction-tools class=headerLink><a href=#prediction-tools class=header-mark></a>Prediction tools</h3><p>However, many current prediction tools lack sufficient accuracy for WC modeling.</p><h3 id=published-models class=headerLink><a href=#published-models class=header-mark></a>Published models</h3><p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5997489/bin/NIHMS928198-supplement.xlsx target=_blank rel="noopener noreffer">A list of public model repos</a> (in Excel file).</p><h3 id=emerging-methods-and-tools-data-aggregation-and-organization class=headerLink><a href=#emerging-methods-and-tools-data-aggregation-and-organization class=header-mark></a>Emerging methods and tools, Data aggregation and organization</h3><p><img class=lazyload data-src=https://user-images.githubusercontent.com/40054455/113999412-9bb36200-988c-11eb-81be-ac4fbc4a6f23.png data-srcset="https://user-images.githubusercontent.com/40054455/113999412-9bb36200-988c-11eb-81be-ac4fbc4a6f23.png, https://user-images.githubusercontent.com/40054455/113999412-9bb36200-988c-11eb-81be-ac4fbc4a6f23.png 1.5x, https://user-images.githubusercontent.com/40054455/113999412-9bb36200-988c-11eb-81be-ac4fbc4a6f23.png 2x" data-sizes=auto alt=https://user-images.githubusercontent.com/40054455/113999412-9bb36200-988c-11eb-81be-ac4fbc4a6f23.png title=Fig2></p><ul><li>(a) Data should be aggregated from thousands of publications, repositories, and prediction tools and organized into a PGDB. (e.g. <a href=https://pubmed.ncbi.nlm.nih.gov/26454094/ target=_blank rel="noopener noreffer">Pathway Tools</a>, <a href=https://pubmed.ncbi.nlm.nih.gov/23175606/ target=_blank rel="noopener noreffer">Whole Cell KB</a>)</li><li>(b) Models should be designed, calibrated, and validated from PGDBs and described using rules.</li><li>(c) Models should be simulated using parallel, network-free, multi-algorithmic simulators and their results should be stored in a database.</li><li>(d) Simulation results should be visualized and analyzed.</li><li>(e) Results should be validated by comparison to experimental measurements. Importantly, all of these steps should be collaborative.</li></ul><h3 id=scalable-model-design class=headerLink><a href=#scalable-model-design class=header-mark></a>Scalable model design</h3><ul><li><a href=https://pubmed.ncbi.nlm.nih.gov/23549147/ target=_blank rel="noopener noreffer">Cell Collective</a></li><li><a href=https://pubmed.ncbi.nlm.nih.gov/22262672/ target=_blank rel="noopener noreffer">MetaFlux</a></li><li><a href=https://pubmed.ncbi.nlm.nih.gov/23423320/ target=_blank rel="noopener noreffer">PySB</a></li><li><a href=https://www.ncbi.nlm.nih.gov/pubmed/26160520/ target=_blank rel="noopener noreffer">SEEK</a></li><li><a href=https://www.ncbi.nlm.nih.gov/pubmed/22139996/ target=_blank rel="noopener noreffer">Virtual Cell</a></li></ul><h3 id=model-languages class=headerLink><a href=#model-languages class=header-mark></a>Model languages</h3><ul><li><a href=https://www.ncbi.nlm.nih.gov/pubmed/12611808/ target=_blank rel="noopener noreffer">SBML</a></li><li><a href=https://www.ncbi.nlm.nih.gov/pubmed/27402907/ target=_blank rel="noopener noreffer">BioNetGen</a></li></ul><h3 id=simulation-tools class=headerLink><a href=#simulation-tools class=header-mark></a>Simulation tools</h3><ul><li><a href=https://www.ncbi.nlm.nih.gov/pubmed/19399433/ target=_blank rel="noopener noreffer">COPASI</a></li><li><a href=https://www.ncbi.nlm.nih.gov/pubmed/22139996/ target=_blank rel="noopener noreffer">Virtual cell</a></li><li><a href=https://www.ncbi.nlm.nih.gov/pubmed/23927696/ target=_blank rel="noopener noreffer">COBRAPy</a></li><li><a href="https://scholar.google.com/scholar_lookup?journal=Rev+Cell+Biol+Mol+Med&title=E-Cell:+Computer+simulation+of+the+cell&author=PK+Dhar&author=K+Takahashi&author=Y+Nakayama&author=M+Tomita&publication_year=2012&" target=_blank rel="noopener noreffer">E-Cell</a></li></ul><h3 id=calibration class=headerLink><a href=#calibration class=header-mark></a>Calibration</h3><ul><li><a href=https://www.ncbi.nlm.nih.gov/pubmed/28109249/ target=_blank rel="noopener noreffer">saCeSS</a> : distributed calibration of large biochemical models</li><li>surrogate models to calibrate large models</li></ul><h3 id=verification class=headerLink><a href=#verification class=header-mark></a>Verification</h3><ul><li><a href="https://scholar.google.com/scholar_lookup?journal=Int+Conf+Comput+Meth+Syst+Biol&title=Statistical+model+checking+in+BioLab:+Applications+to+the+automated+analysis+of+T-cell+receptor+signaling+pathway&author=EM+Clarke&author=JR+Faeder&author=CJ+Langmead&author=LA+Harris&author=SK+Jha&publication_year=2008&pages=231-250&" target=_blank rel="noopener noreffer">BioLab</a></li><li><a href="https://scholar.google.com/scholar_lookup?journal=Computer+Aided+Verification&title=PRISM+4.0:+Verification+of+probabilistic+real-time+systems&author=M+Kwiatkowska&author=G+Norman&author=D+Parker&publication_year=2011&pages=585-591&" target=_blank rel="noopener noreffer">PRISM</a></li></ul><h3 id=simulation-results-analysis class=headerLink><a href=#simulation-results-analysis class=header-mark></a>Simulation results analysis</h3><ul><li>COPASI</li><li>Virtual Cell</li><li>WholeCellSimDB and WholeCellViz</li></ul><h2 id=technological-challenges class=headerLink><a href=#technological-challenges class=header-mark></a>TECHNOLOGICAL CHALLENGES</h2><ul><li>Experimental measurement: Measuring kinetic parameters at the interactome scale.</li><li>Prediction tools: Accurately predict the molecular effects of insertions, deletions, and structural variants.</li><li>Data aggregation</li><li>Scalable, data-driven model design</li><li>Rule-based model representation: No existing language supports all of the biological processes that WC models must represent.</li><li>Scalable multi-algorithmic simulation</li><li>Calibration and verification</li><li>Simulation analysis</li><li>Collaboration</li></ul><h2 id=highlights class=headerLink><a href=#highlights class=header-mark></a>Highlights</h2><ul><li>Whole-cell models predict phenotype from genotype by representing each gene function</li><li>Whole-cell models could transform bioscience, bioengineering, and medicine</li><li>There are many challenges to achieve whole-cell models</li><li>New measurement and modeling technologies are rapidly enabling whole-cell modeling</li><li>Ongoing efforts to build scalable modeling tools will accelerate whole-cell modeling</li></ul><div class=footnotes role=doc-endnotes><hr><ol><li id=fn:1><p>Goldberg AP, Szigeti B, Chew YH, Sekar JA, Roth YD, Karr JR. Emerging whole-cell modeling principles and methods. Curr Opin Biotechnol. 2018;51:97–102.&#160;<a href=#fnref:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li></ol></div></div><div class=post-footer id=post-footer><div class=post-info><div class=post-info-line><div class=post-info-mod><span>Updated on 2022-07-07&nbsp;<a class=git-hash href=https://github.com/sosiristseng/readlist/commit/640e6964e343f4b8efe341b974d3ff1abf09f281 target=_blank title="commit by Wen-Wei Tseng(sosiristseng@gmail.com) 640e6964e343f4b8efe341b974d3ff1abf09f281: move files" rel="noopener noreferrer">
<i class="fas fa-hashtag fa-fw"></i>640e696</a></span></div><div class=post-info-license></div></div><div class=post-info-line><div class=post-info-md><span><a class=link-to-mardown href=/readlist/2021/04/goldberg-2017/index.md target=_blank rel="noopener noreferrer">Read markdown</a>
</span><span>|&nbsp;<a class=link-to-source href=https://github.com/sosiristseng/readlist/blob/main/content/posts/g/goldberg-2017.md target=_blank rel="noopener noreferrer">View source</a>
</span><span>|&nbsp;<a class=link-to-edit href=https://github.com/sosiristseng/readlist/edit/main/content/posts/g/goldberg-2017.md target=_blank rel="noopener noreferrer">Edit this page</a></span></div><div class=post-info-share><span><a href=# onclick=return!1 title="Share on Twitter" data-sharer=twitter data-url=https://sosiristseng.github.io/readlist/2021/04/goldberg-2017/ data-title="Goldberg 2017 : Emerging whole-cell modeling principles and methods" data-via=sosiristseng data-hashtags=ODE><i class="fab fa-twitter fa-fw"></i></a><a href=# onclick=return!1 title="Share on Facebook" data-sharer=facebook data-url=https://sosiristseng.github.io/readlist/2021/04/goldberg-2017/ data-hashtag=ODE><i class="fab fa-facebook-square fa-fw"></i></a><a href=# onclick=return!1 title="Share on Hacker News" data-sharer=hackernews data-url=https://sosiristseng.github.io/readlist/2021/04/goldberg-2017/ data-title="Goldberg 2017 : Emerging whole-cell modeling principles and methods"><i class="fab fa-hacker-news fa-fw"></i></a><a href=# onclick=return!1 title="Share on Reddit" data-sharer=reddit data-url=https://sosiristseng.github.io/readlist/2021/04/goldberg-2017/><i class="fab fa-reddit fa-fw"></i></a><a href=# onclick=return!1 title="Share on Line" data-sharer=line data-url=https://sosiristseng.github.io/readlist/2021/04/goldberg-2017/ data-title="Goldberg 2017 : Emerging whole-cell modeling principles and methods"><i data-svg-src=/readlist/lib/simple-icons/icons/line.min.svg></i></a><a href=# onclick=return!1 title="Share on Telegram" data-sharer=telegram data-url=https://sosiristseng.github.io/readlist/2021/04/goldberg-2017/ data-title="Goldberg 2017 : Emerging whole-cell modeling principles and methods" data-web><i class="fab fa-telegram-plane fa-fw"></i></a></span></div></div></div><div class=post-info-more><section class=post-tags><i class="fas fa-tags fa-fw"></i>&nbsp;<a href=/readlist/tags/ode/>ODE</a></section><section><span><a href=javascript:void(0); onclick=window.history.back()>Back</a></span>&nbsp;|&nbsp;<span><a href=/readlist/>Home</a></span></section></div><div class=post-nav><a href=/readlist/2021/01/gregg-2016/ class=prev rel=prev title="Gregg 2016 : Pancreatic β-Cells From Mice Offset Age-Associated Mitochondrial Deficiency With Reduced KATP Channel Activity"><i class="fas fa-angle-left fa-fw"></i>Gregg 2016 : Pancreatic β-Cells From Mice Offset Age-Associated Mitochondrial Deficiency With Reduced KATP Channel Activity</a>
<a href=/readlist/2021/04/zamponi-2018/ class=next rel=next title="Zamponi 2018 : Mitochondrial network complexity emerges from fission/fusion dynamics">Zamponi 2018 : Mitochondrial network complexity emerges from fission/fusion dynamics<i class="fas fa-angle-right fa-fw"></i></a></div></div></article></div></main><footer class=footer><div class=footer-container><div class=footer-line>Powered by <a href=https://gohugo.io/ target=_blank rel="noopener noreffer" title="Hugo 0.101.0">Hugo</a>&nbsp;|&nbsp;Hosted on <a title="Github Pages" href=https://docs.github.com/en/pages/ target=_blank rel="noopener noreffer">GitHub Pages</a>&nbsp;|&nbsp;Theme - <a href=https://github.com/HEIGE-PCloud/DoIt target=_blank rel="noopener noreffer" title="DoIt 0.2.13"><i class="far fa-edit fa-fw"></i> DoIt</a></div><div class=footer-line><i class="far fa-copyright fa-fw"></i><span itemprop=copyrightYear>2022</span><span class=author itemprop=copyrightHolder>&nbsp;<a href=https://github.com/sosiristseng target=_blank rel="noopener noreferrer">Wen-Wei Tseng</a></span>&nbsp;|&nbsp;<span class=license><a rel="license external nofollow noopener noreferrer" href=https://creativecommons.org/licenses/by-nc/4.0/ target=_blank>CC BY-NC 4.0</a></span></div><div class=footer-line></div><div class=footer-line></div></div><script>"serviceWorker"in navigator&&(navigator.serviceWorker.register("/sw.min.js",{scope:"/"}).then(function(){}),navigator.serviceWorker.ready.then(function(){}))</script></footer></div><div id=fixed-buttons><a href=#back-to-top id=back-to-top-button class=fixed-button title="Back to Top"><i class="fas fa-arrow-up fa-fw"></i>
</a><a href=# id=view-comments class=fixed-button title="View Comments"><i class="fas fa-comment fa-fw"></i></a></div><div class=assets><script type=text/javascript src=/readlist/lib/autocomplete/autocomplete.min.js></script><script type=text/javascript src=/readlist/lib/fuse/fuse.min.js></script><script type=text/javascript src=/readlist/lib/lazysizes/lazysizes.min.js></script><script type=text/javascript src=/readlist/lib/topbar/topbar.min.js></script><script type=text/javascript src=/readlist/lib/pjax/pjax.min.js></script><script type=text/javascript src=/readlist/js/theme.min.js defer></script></div><div class=pjax-assets><script type=text/javascript>window.config={code:{copyTitle:"Copy to clipboard",maxShownLines:15},comment:{},math:{delimiters:[{display:!0,left:"$$",right:"$$"},{display:!0,left:"\\[",right:"\\]"},{display:!1,left:"$",right:"$"},{display:!1,left:"\\(",right:"\\)"}],strict:!1},search:{distance:100,findAllMatches:!1,fuseIndexURL:"/readlist/index.json",highlightTag:"em",ignoreFieldNorm:!1,ignoreLocation:!1,isCaseSensitive:!1,location:0,maxResultLength:10,minMatchCharLength:3,noResultsFound:"No results found",snippetLength:30,threshold:.3,type:"fuse",useExtendedSearch:!1},sharerjs:!0}</script><script type=text/javascript src=/readlist/lib/clipboard/clipboard.min.js></script><script type=text/javascript src=/readlist/lib/sharer/sharer.min.js></script><script type=text/javascript src=/readlist/lib/katex/katex.min.js defer></script><script type=text/javascript src=/readlist/lib/katex/auto-render.min.js defer></script><script type=text/javascript src=/readlist/lib/katex/copy-tex.min.js defer></script><script type=text/javascript src=/readlist/lib/katex/mhchem.min.js defer></script><script type=text/javascript src=/readlist/js/katex.min.js defer></script><link rel=stylesheet href=/readlist/lib/katex/katex.min.css><link rel=preload as=style onload='this.onload=null,this.rel="stylesheet"' href=/readlist/lib/katex/copy-tex.min.css><noscript><link rel=stylesheet href=/readlist/lib/katex/copy-tex.min.css></noscript></div></body></html>